examples/llm/completion/README.md

```diff
@@ -64,9 +66,14 @@ Before running the pipeline, ensure that the `NGC_API_KEY` environment variable
 Install the required dependencies.
 
 ```bash
-mamba env update -n morpheus --file ${MORPHEUS_ROOT}/docker/conda/environments/cuda11.8_examples.yml
+export CUDA_VER=11.8
+mamba install -n base -c conda-forge conda-merge
+conda run -n base --live-stream conda-merge docker/conda/environments/cuda${CUDA_VER}_dev.yml \
+  docker/conda/environments/cuda${CUDA_VER}_examples.yml > .tmp/merged.yml \
+  && mamba env update -n ${CONDA_DEFAULT_ENV} --file .tmp/merged.yml
```

Comment:
I'm still having same conflicts when run this in dev container. The merged yaml only seems to work for me when I use it with `mamba env create`.

Reply:
I also ran this in my host conda environment. I'm getting this error now with most conda/mamba commands:
```
ImportError: libffi.so.7: cannot open shared object file: No such file or directory
thread '<unnamed>' panicked at /home/conda/feedstock_root/build_artifacts/cryptography-split_1698192008573/_build_env/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pyo3-0.18.3/src/err/mod.rs:790:5:
Python API call failed
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Traceback (most recent call last):
  File "/home/efajardo/miniconda3/bin/mamba", line 7, in <module>
    from mamba.mamba import main
  File "/home/efajardo/miniconda3/lib/python3.9/site-packages/mamba/mamba.py", line 22, in <module>
    from conda.cli.install import check_prefix, clone, get_revision
  File "/home/efajardo/miniconda3/lib/python3.9/site-packages/conda/cli/install.py", line 19, in <module>
    from ..core.index import calculate_channel_urls, get_index
  File "/home/efajardo/miniconda3/lib/python3.9/site-packages/conda/core/index.py", line 15, in <module>
    from .subdir_data import SubdirData, make_feature_record
  File "/home/efajardo/miniconda3/lib/python3.9/site-packages/conda/core/subdir_data.py", line 54, in <module>
    from conda_content_trust.common import (
  File "/home/efajardo/miniconda3/lib/python3.9/site-packages/conda_content_trust/common.py", line 63, in <module>
    import cryptography.hazmat.primitives.asymmetric.ed25519 as ed25519
  File "/home/efajardo/miniconda3/lib/python3.9/site-packages/cryptography/hazmat/primitives/asymmetric/ed25519.py", line 9, in <module>
    from cryptography.exceptions import UnsupportedAlgorithm, _Reasons
  File "/home/efajardo/miniconda3/lib/python3.9/site-packages/cryptography/exceptions.py", line 9, in <module>
    from cryptography.hazmat.bindings._rust import exceptions as rust_exceptions
pyo3_runtime.PanicException: Python API call failed
```

Reply:
in release container:
```
(morpheus) root@EFAJARDO-DT:/workspace# conda run -n base --live-stream conda-merge docker/conda/environments/cuda${CUDA_VER}_dev.yml   docker/conda/environments/cuda${CUDA_VER}_examples.yml > .tmp/merged.yml   && mamba env update -n ${CONDA_DEFAULT_ENV} --file .tmp/merged.yml
nvidia/linux-64                                             Using cache
nvidia/noarch                                               Using cache
conda-forge/linux-64                                        Using cache
conda-forge/noarch                                          Using cache
rapidsai/linux-64                                             No change
nvidia/label/cuda-11.8.0/noarch                    135.0 B @ 671.0 B/s  0.2s
rapidsai/noarch                                               No change
nvidia/label/cuda-11.8.0/linux-64                    4.0kB @  16.8kB/s  0.2s
nvidia/label/dev/linux-64                           45.2kB @ 168.6kB/s  0.3s
huggingface/linux-64                                 5.0kB @  18.2kB/s  0.1s
dglteam/label/cu118/linux-64                         2.4kB @   8.3kB/s  0.1s
pytorch/noarch                                      10.7kB @  35.5kB/s  0.1s
pytorch/linux-64                                   178.0kB @ 522.7kB/s  0.1s
dglteam/label/cu118/noarch                         130.0 B @ 362.0 B/s  0.1s
huggingface/noarch                                  13.1kB @  36.0kB/s  0.1s
nvidia/label/dev/noarch                            130.0 B @ 114.0 B/s  0.9s


Looking for: ['arxiv=1.4', 'automake=1.16.5', 'beautifulsoup4=4.12', 'benchmark=1.6.1', 'boost-cpp=1.82', 'boto3', 'cachetools=5.0.0', "ccache[version='>=3.7']", 'clangdev=16', "click[version='>=8']", 'cmake=3.24', 'configargparse=1.5', 'cuda-compiler=11.8', 'cuda-nvml-dev=11.8', "cuda-python[version='>=11.8,<11.8.3']", 'cuda-toolkit=11.8', 'cudf=23.06', 'cuml=23.06', "cupy[version='>=12.0.0']", 'cxx-compiler', 'cython=0.29.24', "dask[version='>=2023.1.1']", 'datacompy=0.8', 'dgl=1.0.2', 'dill', 'dill=0.3.6', "distributed[version='>=2023.1.1']", 'docker-py=5.0', 'docutils', 'elasticsearch==8.9.0', 'environs=9.5', 'faker=12.3.0', 'feedparser=6.0.10', 'flake8', 'flatbuffers=2.0', 'gcc_linux-64=11.2', 'gflags=2.2', 'git-lfs=3.2', "git[version='>=2.35.3']", 'glog=0.6', "gmock[version='>=1.13.0']", 'gputil', 'grpcio', "gtest[version='>=1.13.0']", 'gxx_linux-64=11.2', 'huggingface_hub=0.10.1', 'include-what-you-use=0.20', 'ipywidgets', 'isort', "jupyter_core[version='>=4.11.2,<5.0']", 'jupyterlab', 'kafka-python=2.0', 'langchain=0.0.190', "libgrpc[version='>=1.49']", 'librdkafka=1.9.2', "libwebp[version='>=1.3.2']", 'minio=7.1', "mlflow[version='>=2.2.1,<3']", 'mrc=23.11', "networkx[version='>=2.8']", 'newspaper3k=0.2', 'ninja=1.10', 'nodejs=18', "numba[version='>=0.56.2']", 'numpydoc=1.4', 'nvtabular=23.06', 'openai=0.28', 'pandas=1.3', 'papermill=2.3.4', 'pip', 'pkg-config', 'pluggy=1.0', 'port-for=0.7', 'protobuf=4.21', 'py4j=0.10', 'pyarrow=[build=*_cuda]', 'pybind11-stubgen=0.10.5', 'pydot', "pylint[version='>=2.17.4,<2.18']", 'pypdf=3.16', 'pytest', 'pytest-asyncio', "pytest-benchmark[version='>=4.0']", 'pytest-cov', 'python-confluent-kafka=1.9.2', 'python-dotenv=1.0', 'python-graphviz', 'python=3.10', 'pytorch-cuda', 'pytorch=2.0.1', 'rapidjson=1.1.0', 'requests-cache=1.1', 'requests=2.31', "s3fs[version='>=2023.6']", 'scikit-build=0.17.1', 'scikit-learn=1.2.2', 'sentence-transformers', 'sphinx', 'sphinx_rtd_theme', "sqlalchemy[version='<2.0']", 'sysroot_linux-64=2.17', 'tqdm=4', 'transformers', 'tritonclient=2.26', 'typing_utils=0.1', 'ujson=5.8', 'watchdog=2.1', 'websockets', 'yapf=0.40.1']


Could not solve for environment specs
The following package could not be installed
└─ langchain 0.0.190**  is installable and it requires
   └─ networkx >=2.6.3,<3.0.0 , which can be installed.
```

Reply:
Should be resolve now; adding the default channel explicitly in each environment yml.

Reply:
@drobison00 We need to change `networkx` in meta.yml to `networkx >=2.8` so it matches whats in the conda env.

Reply:
I still can't add the example dependencies. Getting this in both dev and release containers (freshly built) in two separate environments.

<details><summary>Click here to see error details</summary><pre>

Could not solve for environment specs
The following packages are incompatible
├─ blas-devel is installable with the potential options
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.15 blis, which can be installed;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.15 mkl, which requires
│  │     └─ libblas 3.8.0 15_mkl, which requires
│  │        └─ liblapack 3.8.0 15_mkl, which can be installed;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.15 openblas, which conflicts with any installable versions previously reported;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.16 blis, which conflicts with any installable versions previously reported;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.16 mkl, which requires
│  │     └─ libblas 3.8.0 16_mkl, which requires
│  │        └─ liblapack 3.8.0 16_mkl, which can be installed;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.16 openblas, which conflicts with any installable versions previously reported;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.17 openblas, which conflicts with any installable versions previously reported;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.18 mkl, which requires
│  │     └─ libblas 3.8.0 18_mkl, which requires
│  │        └─ liblapack 3.8.0 18_mkl, which can be installed;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.19 mkl, which requires
│  │     └─ libblas 3.8.0 19_mkl, which requires
│  │        └─ liblapack 3.8.0 19_mkl, which can be installed;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.20 mkl, which requires
│  │     └─ libblas 3.8.0 20_mkl, which requires
│  │        └─ liblapack 3.8.0 20_mkl, which can be installed;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.21 mkl, which requires
│  │     └─ libblas 3.8.0 21_mkl, which requires
│  │        └─ liblapack 3.8.0 21_mkl, which can be installed;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.14 blis, which can be installed;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.14 mkl, which requires
│  │     └─ libblas 3.8.0 14_mkl, which requires
│  │        └─ liblapack 3.8.0 14_mkl, which can be installed;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.14 openblas, which can be installed;
│  ├─ blas-devel 3.8.0 would require
│  │  └─ blas 2.7 netlib, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.0 blis, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.0 openblas, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ libcblas [3.9.0 10_blis|3.9.0 11_linux64_blis|...|3.9.0 9_blis], which requires
│  │     └─ blas * blis, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ mkl >=2021.3.0,<2022.0a0 , which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ liblapacke [3.9.0 10_openblas|3.9.0 11_linux64_openblas|...|3.9.0 9_openblas], which requires
│  │     └─ blas * openblas, which conflicts with any installable versions previously reported;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ mkl >=2021.4.0,<2022.0a0 , which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ mkl-devel 2022.0.* , which requires
│  │     └─ mkl 2022.0.1 h8d4b97c_803, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ mkl-devel 2022.1.* , which requires
│  │     └─ mkl 2022.1.0 h84fe81f_915, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.1 blis, which does not exist (perhaps a missing channel);
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.1 openblas, which does not exist (perhaps a missing channel);
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.2 blis, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.2 openblas, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.3 blis, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.3 openblas, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.4 blis, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.4 mkl, which requires
│  │     └─ libblas 3.8.0 4_mkl, which requires
│  │        └─ mkl 2019.1.* , which does not exist (perhaps a missing channel);
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas [2.4 openblas|2.5 openblas], which requires
│  │     └─ libblas [3.8.0 4_openblas|3.8.0 5_openblas], which requires
│  │        └─ openblas 0.3.5.* , which does not exist (perhaps a missing channel);
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.5 blis, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.5 mkl, which requires
│  │     └─ libblas 3.8.0 5_mkl, which requires
│  │        └─ liblapack 3.8.0 5_mkl, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ libblas 3.9.0 5_h92ddd45_netlib, which requires
│  │     └─ blas * netlib, which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.106 blis, which conflicts with any installable versions previously reported;
│  ├─ blas-devel 3.9.0 would require
│  │  ├─ blas 2.106 mkl, which requires
│  │  │  └─ libblas [3.9.0 5_mkl|3.9.0 6_mkl|3.9.0 7_mkl|3.9.0 8_mkl], which requires
│  │  │     └─ mkl >=2020.4,<2021.0a0 , which can be installed;
│  │  └─ mkl >=2020.4,<2021.0a0 , which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.106 openblas, which conflicts with any installable versions previously reported;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ mkl >=2021.2.0,<2022.0a0 , which can be installed;
│  ├─ blas-devel 3.9.0 would require
│  │  └─ blas 2.0 netlib, which can be installed;
│  └─ blas-devel 3.9.0 would require
│     └─ mkl >=2023.2.0,<2024.0a0 , which can be installed;
├─ cuda-compiler 11.8**  is requested and can be installed;
├─ cuml 23.06**  is installable with the potential options
│  ├─ cuml 23.06.00 would require
│  │  ├─ pylibraft 23.06.*  with the potential options
│  │  │  ├─ pylibraft [23.06.00|23.06.01|23.06.02] would require
│  │  │  │  └─ numpy [>=1.24.3,<2.0a0 |>=1.24.4,<2.0a0 ] with the potential options
│  │  │  │     ├─ numpy [1.24.3|1.24.4|...|1.26.2] would require
│  │  │  │     │  └─ liblapack >=3.9.0,<4.0a0  with the potential options
│  │  │  │     │     ├─ liblapack 3.9.0 conflicts with any installable versions previously reported;
│  │  │  │     │     ├─ liblapack 3.9.0 would require
│  │  │  │     │     │  └─ blas * openblas, which conflicts with any installable versions previously reported;
│  │  │  │     │     ├─ liblapack 3.9.0 would require
│  │  │  │     │     │  └─ libblas [3.9.0 10_mkl|3.9.0 11_linux64_mkl], which requires
│  │  │  │     │     │     └─ mkl >=2021.3.0,<2022.0a0 , which can be installed;
│  │  │  │     │     ├─ liblapack 3.9.0 would require
│  │  │  │     │     │  └─ libblas 3.9.0 12_linux64_mkl, which requires
│  │  │  │     │     │     └─ mkl >=2021.4.0,<2022.0a0 , which can be installed;
│  │  │  │     │     ├─ liblapack 3.9.0 would require
│  │  │  │     │     │  └─ libblas [3.9.0 4_mkl|3.9.0 5_mkl|3.9.0 6_mkl|3.9.0 7_mkl|3.9.0 8_mkl], which can be installed (as previously explained);
│  │  │  │     │     ├─ liblapack 3.9.0 would require
│  │  │  │     │     │  └─ libblas 3.9.0 9_mkl, which requires
│  │  │  │     │     │     └─ mkl >=2021.2.0,<2022.0a0 , which can be installed;
│  │  │  │     │     └─ liblapack 3.9.0 would require
│  │  │  │     │        └─ libblas [3.9.0 19_linux64_mkl|3.9.0 20_linux64_mkl], which requires
│  │  │  │     │           └─ mkl >=2023.2.0,<2024.0a0 , which can be installed;
│  │  │  │     ├─ numpy [1.24.3|1.24.4|...|1.26.2] would require
│  │  │  │     │  └─ python_abi 3.11.* *_cp311, which can be installed;
│  │  │  │     ├─ numpy [1.24.3|1.24.4] would require
│  │  │  │     │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  │  │  │     ├─ numpy [1.24.3|1.24.4] would require
│  │  │  │     │  └─ pypy3.8 >=7.3.11 , which can be installed;
│  │  │  │     ├─ numpy [1.24.3|1.24.4|1.25.0|1.25.1|1.25.2] would require
│  │  │  │     │  └─ pypy3.9 >=7.3.11 , which can be installed;
│  │  │  │     ├─ numpy [1.24.3|1.24.4|...|1.26.2] would require
│  │  │  │     │  └─ python >=3.9,<3.10.0a0 , which can be installed;
│  │  │  │     ├─ numpy 1.26.0 would require
│  │  │  │     │  └─ python >=3.12.0rc3,<3.13.0a0  with the potential options
│  │  │  │     │     ├─ python 3.12.0 would require
│  │  │  │     │     │  └─ python_abi 3.12.* *_cp312, which can be installed;
│  │  │  │     │     └─ python 3.12.0rc3 would require
│  │  │  │     │        └─ _python_rc, which does not exist (perhaps a missing channel);
│  │  │  │     ├─ numpy 1.26.0 would require
│  │  │  │     │  └─ pypy3.9 >=7.3.12 , which can be installed;
│  │  │  │     ├─ numpy 1.26.2 would require
│  │  │  │     │  └─ python >=3.12,<3.13.0a0 , which can be installed (as previously explained);
│  │  │  │     └─ numpy 1.26.2 would require
│  │  │  │        └─ pypy3.9 >=7.3.13 , which can be installed;
│  │  │  └─ pylibraft [23.06.00|23.06.01|23.06.02] would require
│  │  │     └─ python >=3.9,<3.10.0a0 , which can be installed;
│  │  ├─ python >=3.10,<3.11.0a0  but there are no viable options
│  │  │  ├─ python 3.10.8 would require
│  │  │  │  └─ graalpy >=23.0.0,<23.0.1.0a0 , which requires
│  │  │  │     └─ python_abi 3.10.* *_graalpy230_310_native, which can be installed;
│  │  │  └─ python [3.10.0|3.10.1|...|3.10.9] would require
│  │  │     └─ python_abi 3.10.* *_cp310, which conflicts with any installable versions previously reported;
│  │  └─ python_abi 3.10.* *_cp310, which conflicts with any installable versions previously reported;
│  └─ cuml 23.06.00 would require
│     ├─ python >=3.9,<3.10.0a0 , which can be installed;
│     └─ python_abi 3.9.* *_cp39, which can be installed;
├─ cython 0.29.24**  is installable with the potential options
│  ├─ cython 0.29.24 would require
│  │  ├─ python >=3.10,<3.11.0a0 , which cannot be installed (as previously explained);
│  │  └─ python_abi 3.10.* *_cp310, which conflicts with any installable versions previously reported;
│  ├─ cython 0.29.24 would require
│  │  └─ python >=3.6,<3.7.0a0 , which can be installed;
│  ├─ cython 0.29.24 would require
│  │  └─ python >=3.7,<3.8.0a0 , which can be installed;
│  ├─ cython 0.29.24 would require
│  │  └─ python_abi 3.7 *_pypy37_pp73, which requires
│  │     └─ python 3.7.* *_73_pypy, which can be installed;
│  ├─ cython 0.29.24 would require
│  │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  └─ cython 0.29.24 would require
│     └─ python >=3.9,<3.10.0a0 , which can be installed;
├─ langchain 0.0.190**  is installable and it requires
│  └─ numexpr >=2.8.4,<3.0.0  with the potential options
│     ├─ numexpr [2.8.4|2.8.7] would require
│     │  └─ mkl >=2022.2.1,<2023.0a0 , which conflicts with any installable versions previously reported;
│     ├─ numexpr [2.8.4|2.8.7] would require
│     │  └─ python_abi 3.11.* *_cp311, which can be installed;
│     ├─ numexpr 2.8.4 would require
│     │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│     ├─ numexpr 2.8.4 would require
│     │  └─ pypy3.8 >=7.3.11 , which can be installed;
│     ├─ numexpr 2.8.4 would require
│     │  └─ pypy3.9 >=7.3.11 , which can be installed;
│     ├─ numexpr [2.8.4|2.8.7] would require
│     │  └─ python >=3.9,<3.10.0a0 , which can be installed;
│     ├─ numexpr [2.8.4|2.8.7] would require
│     │  └─ nomkl, which requires
│     │     └─ mkl <0.a0 , which conflicts with any installable versions previously reported;
│     ├─ numexpr 2.8.7 would require
│     │  └─ python >=3.12,<3.13.0a0 , which can be installed (as previously explained);
│     ├─ numexpr 2.8.7 would require
│     │  └─ pypy3.9 >=7.3.13 , which can be installed;
│     ├─ numexpr 2.8.7 would require
│     │  └─ python >=3.12.0rc3,<3.13.0a0  with the potential options
│     │     ├─ python 3.12.0, which can be installed (as previously explained);
│     │     └─ python 3.12.0rc3, which cannot be installed (as previously explained);
│     └─ numexpr 2.8.7 would require
│        └─ pypy3.9 >=7.3.12 , which can be installed;
├─ morpheus is not installable because it requires
│  ├─ cuda-cudart >=11.8,<12.0a0 , which can be installed;
│  └─ python >=3.10,<3.11.0a0 , which cannot be installed (as previously explained);
└─ pytorch 2.0.1**  is installable with the potential options
   ├─ pytorch 2.0.1 would require
   │  └─ blas * mkl with the potential options
   │     ├─ blas [1.0|2.10|...|2.9] conflicts with any installable versions previously reported;
   │     ├─ blas [2.105|2.106|2.107|2.108], which can be installed (as previously explained);
   │     ├─ blas 2.109 would require
   │     │  └─ libblas 3.9.0 9_mkl, which can be installed (as previously explained);
   │     ├─ blas [2.110|2.111] would require
   │     │  └─ libblas [3.9.0 10_mkl|3.9.0 11_linux64_mkl], which can be installed (as previously explained);
   │     ├─ blas 2.112 would require
   │     │  └─ libblas 3.9.0 12_linux64_mkl, which can be installed (as previously explained);
   │     ├─ blas [2.113|2.114] would require
   │     │  └─ blas-devel [3.9.0 13_linux64_mkl|3.9.0 14_linux64_mkl], which can be installed (as previously explained);
   │     ├─ blas [2.115|2.116] would require
   │     │  └─ blas-devel [3.9.0 15_linux64_mkl|3.9.0 16_linux64_mkl], which can be installed (as previously explained);
   │     ├─ blas 2.14, which cannot be installed (as previously explained);
   │     ├─ blas 2.15, which can be installed (as previously explained);
   │     ├─ blas 2.16, which can be installed (as previously explained);
   │     ├─ blas 2.18, which can be installed (as previously explained);
   │     ├─ blas 2.19, which can be installed (as previously explained);
   │     ├─ blas 2.20, which can be installed (as previously explained);
   │     ├─ blas 2.21, which can be installed (as previously explained);
   │     ├─ blas 2.4, which cannot be installed (as previously explained);
   │     ├─ blas 2.5, which cannot be installed (as previously explained);
   │     └─ blas [2.119|2.120] would require
   │        └─ libblas [3.9.0 19_linux64_mkl|3.9.0 20_linux64_mkl], which can be installed (as previously explained);
   ├─ pytorch 2.0.1 would require
   │  └─ pytorch-cuda >=11.7,<11.8  but there are no viable options
   │     ├─ pytorch-cuda 11.7 would require
   │     │  └─ cuda-cudart >=11.7,<11.8 , which conflicts with any installable versions previously reported;
   │     └─ pytorch-cuda 11.7 would require
   │        └─ cuda-compiler >=11.7,<11.8 , which conflicts with any installable versions previously reported;
   ├─ pytorch 2.0.1 would require
   │  └─ python >=3.11,<3.12.0a0 , which can be installed;
   ├─ pytorch 2.0.1 would require
   │  └─ python >=3.8,<3.9.0a0 , which can be installed;
   └─ pytorch 2.0.1 would require
      └─ python >=3.9,<3.10.0a0 , which can be installed.

</pre></details>

Reply:
works now. i was able to add dependencies to dev/release containers and conda environment on host.